helix-term/src/ui/completion.rs

```diff
@@ -26,54 +34,63 @@ impl menu::Item for CompletionItem {
-                Some(lsp::CompletionItemKind::CONSTANT) => "constant",
-                Some(lsp::CompletionItemKind::STRUCT) => "struct",
-                Some(lsp::CompletionItemKind::EVENT) => "event",
-                Some(lsp::CompletionItemKind::OPERATOR) => "operator",
-                Some(lsp::CompletionItemKind::TYPE_PARAMETER) => "type_param",
-                Some(kind) => unimplemented!("{:?}", kind),
-                None => "",
-            }),
-            // self.detail.as_deref().unwrap_or("")
-            // self.label_details
-            //     .as_ref()
-            //     .or(self.detail())
-            //     .as_str(),
+            menu::Cell::from(self.label(data)),
+            match self {
+                CompletionItem::LSP { item, .. } => {
+                    menu::Cell::from(match item.kind {
+                        Some(lsp::CompletionItemKind::TEXT) => "text",
+                        Some(lsp::CompletionItemKind::METHOD) => "method",
+                        Some(lsp::CompletionItemKind::FUNCTION) => "function",
+                        Some(lsp::CompletionItemKind::CONSTRUCTOR) => "constructor",
+                        Some(lsp::CompletionItemKind::FIELD) => "field",
+                        Some(lsp::CompletionItemKind::VARIABLE) => "variable",
+                        Some(lsp::CompletionItemKind::CLASS) => "class",
+                        Some(lsp::CompletionItemKind::INTERFACE) => "interface",
+                        Some(lsp::CompletionItemKind::MODULE) => "module",
+                        Some(lsp::CompletionItemKind::PROPERTY) => "property",
+                        Some(lsp::CompletionItemKind::UNIT) => "unit",
+                        Some(lsp::CompletionItemKind::VALUE) => "value",
+                        Some(lsp::CompletionItemKind::ENUM) => "enum",
+                        Some(lsp::CompletionItemKind::KEYWORD) => "keyword",
+                        Some(lsp::CompletionItemKind::SNIPPET) => "snippet",
+                        Some(lsp::CompletionItemKind::COLOR) => "color",
+                        Some(lsp::CompletionItemKind::FILE) => "file",
+                        Some(lsp::CompletionItemKind::REFERENCE) => "reference",
+                        Some(lsp::CompletionItemKind::FOLDER) => "folder",
+                        Some(lsp::CompletionItemKind::ENUM_MEMBER) => "enum_member",
+                        Some(lsp::CompletionItemKind::CONSTANT) => "constant",
+                        Some(lsp::CompletionItemKind::STRUCT) => "struct",
+                        Some(lsp::CompletionItemKind::EVENT) => "event",
+                        Some(lsp::CompletionItemKind::OPERATOR) => "operator",
+                        Some(lsp::CompletionItemKind::TYPE_PARAMETER) => "type_param",
+                        Some(kind) => unimplemented!("{:?}", kind),
+                        None => "",
+                    })
+                    // self.detail.as_deref().unwrap_or("")
```

Comment:
Still needed?

Reply:
This was commented out already, and I just left it in, but I agree and removed it in the last commit.